.github/delete-azure-resources.sh

```diff
@@ -3,9 +3,10 @@
 # The following environment variables need to be configured before running the script
 # - RESOURCE_GROUP_NAME
 # - STORAGE_ACCOUNT_NAME
-# - APP_CONFIG_NAME
+# - KEY_VAULT_NAME
 
 az appconfig delete --name ${APP_CONFIG_NAME} --resource-group ${RESOURCE_GROUP_NAME} --yes
 az appconfig purge --name ${APP_CONFIG_NAME} --yes
 az storage account delete --name ${STORAGE_ACCOUNT_NAME} --resource-group ${RESOURCE_GROUP_NAME} --yes
+az keyvault delete --name ${KEY_VAULT_NAME} --resource-group ${RESOURCE_GROUP_NAME} --yes
 az group delete --name ${RESOURCE_GROUP_NAME} --yes
```

Comment:
Suggest to purge the azure keyvault to allow the pipeline to rerun which will create the keyvault with the same name. See https://github.com/Azure/azure-microprofile/blob/main/.github/build-with-azure-resources.sh#L39-L53

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/1b0f9b4ce9d8e8cfba02237c818ba1bdb4c90574.